block_additional_mappings_intro.md

```diff
@@ -48,3 +48,12 @@ Block Additional Mappings define the `BlockAddIDType` value reserved to identify
 When the Block Additional Mapping is dependent on additional contextual information then the Mapping SHOULD describe how such additional contextual information is stored within the `BlockAddIDExtraData Element`.
 
 The following Block Additional Mappings are defined.
+
+## Summary of Assigned BlockAddIDType Values
+
+For convenience, the following table shows the assigned BlockAddIDType values along with the BlockAddIDName and Citation.
+
+| BlockAddIDType | BlockAddIDName                                               | Citation                             |
+|:---------------|:-------------------------------------------------------------|:-------------------------------------|
+| 12             | SMPTE ST 12-1 timecode                                       | (#smpte-st12-1-timecode)             |
```

Comment:
Should be 121 according to #348.

If the goal is to add just the Key Value Pair after SMPTE 12-1, rebase this branch on top of  `add-timecode-block-additional-mapping`.

Reply:
Yes, this pr is built upon https://github.com/cellar-wg/matroska-specification/pull/348 (which I suggest to resolve first). I rebased so now it matches 348.